starter/app.js

```diff
@@ -0,0 +1,125 @@
+
+//--------------------------------BASICS-----------------------------------------
+
+// function Hello(){
+//     const name = "NodeJS";
+//     console.log(name);
+//     console.log(7+3);
+// }
+// Hello();
+
+//------------------------------BASICS--------------------------------------------
+
+//blocking Synchronous way FILE-----------------------------------------------------
+
+//const fs= require('fs');
+
+// const textIn= fs.readFileSync('./txt/input.txt', 'utf-8');
+// console.log(textIn);
+
+//-----------------------------------------------------------------------------
+// const fs= require('fs');
+// const textOut = `this are all avacados: ${textIn}\n created at: ${Date.now()}`;
+// fs.writeFileSync('./txt/output.txt', textOut);
+// console.log('hai hello')
+
+//------------------------------------------------------------------------------
+
+// Non blocking Synchronous way
+
+// fs.readFile('./txt/start.txt', 'utf-8', (err, data) =>{
+// console.log(data);
+// });
+// console.log('will read file')   
+
+//------------------------------------------------------------------------------------------
+
+//-------------------------Server-----------------------------------------------------------
+
+// const http = require ('http');
+//  const Server =http.createServer((req, res) => {
+//     console.log(req)
+//     res.end('hello welcome to the server side');
+
+// });
+// Server.listen('8900', '127.0.0.1', () =>{
+//     console.log('Listnening to request on port  8000')
+// })
+
+
```

Comment:
Don't commit commented code in pr

Reply:
I don't think this is a correct flow. kindly refer these github projects, you may follow Udemy course but also refer this as a example

https://github.com/nnance/express-ejs-html
https://github.com/maitraysuthar/rest-api-nodejs-mongodb

Reply:
> Don't commit commented code in pr


Sure. Hereafter, I will clean up commented code before check-in.